.fx/subscriptionInfo.json

```diff
@@ -0,0 +1,5 @@
+{
+  "subscriptionId": "0f16d84b-3c5e-4cfa-be32-a78c3329897b",
+  "subscriptionName": "Azure for Students",
+  "tenantId": "ce6d05e1-3c5e-4d62-87a8-4c4a2713c113"
+}
```

Comment:
Was this an intended change for the PR, or is this a side effect of Azure doing it's thing?

Reply:
Azure keeps adding it. It's actually from the other branch but everytime I open vscode it reads it.

Reply:
Good point though I'll delete  from this branch

Reply:
deleted.